cmd/notation/cert.go

```diff
@@ -440,8 +457,8 @@ func printCerts(root string) error {
 	})
 }
 
-// showRootCA displays details of a root certificate
-func showRootCA(cert *x509.Certificate) {
+// showCert displays details of a certificate
+func showCert(cert *x509.Certificate) {
```

Comment:
Do we need to use `tabwriter` to print the result?  For example: `ioutil.PrintPlugins`

Reply:
Interesting, didn't know we have this. Let me try!

Reply:
I found that tabWriter writes horizontally, which does not work very well for long fields such as cert.PublicKey.  Will stick to vertical list format.